### PR TITLE
BROKERS: Change Task ForeignKey Configuration

### LINKED ISSUE #354: BROKERS: Change Task ForeignKey  Configuration

### PR DESCRIPTION
**Pull request by MirzayevDeveloper**
Closes #354

### CODE REVIEW

File: Tarteeb.Api/Models/Foundations/Tickets/Ticket.cs [hunk 1]
```diff
@@ -22,19 +23,19 @@ public class Ticket
         public DateTimeOffset UpdatedDate { get; set; }
 
         public Guid? AssigneeId { get; set; }
-        [JsonIgnore]
+        [NotMapped]
         public User Assignee { get; set; }
 
         public Guid CreatedUserId { get; set; }
-        [JsonIgnore]
+        [NotMapped]
```
Comment (ElbekDeveloper) [inline Tarteeb.Api/Models/Foundations/Tickets/Ticket.cs:30]: Nima uchun JsonIgnore o'rniga buni ishlatyapmiz

Comment (MirzayevDeveloper) [reply Tarteeb.Api/Models/Foundations/Tickets/Ticket.cs:30]: Unable to determine the relationship represented by navigation 'Ticket.CreatedUser' of type 'User'. Either manually configure the relationship, or ignore this property using the '[NotMapped]' attribute or by using 'EntityTypeBuilder.Ignore' in 'OnModelCreating'. 

NotMapped usulidan foydalandim

Comment (ElbekDeveloper) [reply Tarteeb.Api/Models/Foundations/Tickets/Ticket.cs:30]: Ikkinchisidan foydalanishingiz kerak.

**Comment by ElbekDeveloper** [comment]
@iamDiyorjon PR lari buni hal qilgani uchun bu PR yopiladi

**Pull request closed without merging**